website/layouts/partials/_default/banner.html

```diff
@@ -0,0 +1,16 @@
+{{- $currentVersion := index (split .Dir "/") 1 }}
+{{- $latestVersion := .Site.Params.latest }}
```

Comment:
I see one inconsistence. We sometimes refer to `tip` through variable `Site.Params.latest` sometimes, hardcoded `tip`. Let's decide on one (:

Reply:
I think hardcoded is best approach. Reflecting changes across pages of similar occurence